site/en/guides/build/commissioning.md

```diff
@@ -23,24 +23,24 @@ For an example of commissioning using virtual devices, see the
 
 ## Step 1: Enable roles
 
-To enable the Commissioner and Joiner roles, use the following build switches:
+To enable the Commissioner and Joiner roles, use the following cmake options:
 
-Switch | Description
+Option | Description
 ---- | ----
-`COMMISSIONER=1` | Enables the Commissioner role
-`JOINER=1` | Enables the Joiner role
+`-DOT_COMMISSIONER=ON` | Enables the Commissioner role
+`-DOT_JOINER=ON` | Enables the Joiner role
 
 For example, to build the CC2538 example platform for use as a Joiner only:
 
 ```
-$ make -f examples/Makefile-cc2538 JOINER=1
+$ ./script/build -DOT_JOINER=ON
```

Comment:
Which repository are we referring to here?

Reply:
https://github.com/openthread/ot-cc2538

I didn't mention it on this page since the it was already covered in `index.md`.